WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardReadersView.swift

```diff
@@ -0,0 +1,80 @@
+import SwiftUI
+import WebKit
+import SafariServices
+
+struct SafariView: UIViewControllerRepresentable {
+
+    var choice: Manual
+    let url: URL
+
+    func makeUIViewController(context: UIViewControllerRepresentableContext<SafariView>) -> SFSafariViewController {
+        return SFSafariViewController(url: url)
+    }
+
+    func updateUIViewController(_ uiViewController: SFSafariViewController,
+                                context: UIViewControllerRepresentableContext<SafariView>) {
+
+    }
+}
+/// A view to be displayed on Card Reader Manuals screen
+///
+struct CardReadersView: View {
+    // Tracks the scale of the view due to accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    let viewModel = CardReadersViewViewModel()
+    var manuals: [Manual] {
+        viewModel.manuals
+    }
+
+    var body: some View {
+        GeometryReader { geometry in
+            ScrollView {
+                VStack(alignment: .leading, spacing: 0) {
+                    ForEach(manuals, id: \.name) { manual in
+                        Divider()
+                        NavigationLink(destination: SafariView(choice: manual, url: URL(string: manual.urlString)!)) {
+                                Image(uiImage: manual.image)
+                                    .resizable()
+                                    .aspectRatio(contentMode: .fit)
+                                    .frame(width: Constants.imageSize * scale, height: Constants.imageSize * scale, alignment: .center)
+                                    .frame(width: geometry.size.width * Constants.imageSizeMultiplier)
+                                Text(manual.name)
+                                .frame(width: geometry.size.width * Constants.textSizeMultiplier, alignment: .leading)
+                                .font(.body)
+                                DisclosureIndicator()
+                                .frame(width: geometry.size.width * Constants.imageSizeMultiplier)
+                        }
+                        .frame(height: geometry.size.height * Constants.numberOfRowsMultiplier)
```

Comment:
This may be what's causing the small rows for the manuals. Ideally, we wouldn't need to specify a height for a row based on its container, but let it self-size based on its content.

Reply:
That makes sense! I'll remove the height multiplier and increase the content's Image instead to make the rows bigger, as it also addresses [this comment](https://github.com/woocommerce/woocommerce-ios/pull/7199/files#r913680459)  👍